autogen/cluster_zonal.tf

```diff
@@ -114,6 +114,7 @@ resource "google_container_cluster" "zonal_primary" {
 {% endif %}
 
   remove_default_node_pool = "${var.remove_default_node_pool}"
+  database_encryption      = "${var.database_encryption}"
```

Comment:
Given that this should be a list, can we wrap it with `[]`? e.g. `database_encryption = ["${var.database_encryption}"]`

Reply:
done